src/js/components/hooks/useDebouncedEffect.js

```diff
@@ -0,0 +1,19 @@
+/* @flow */
+import {useEffect, useRef} from "react"
+
+export default function useDebouncedEffect(
```

Comment:
Oh awesome! I was just thinking about adding a debouncer for some of the keyboard events we are handling (up and down arrows). 👍

Reply:
Right on! Lodash has a debounce and throttle utility as well, in case you just need a function debounced.